packages/tinacms/src/index.ts

```diff
@@ -63,10 +59,6 @@ export type TinaCloudCollection = Collection
  * @deprecated use `Collection` instead
  */
 export type TinaCollection = Collection
-/**
- * @deprecated use `Schema` instead
- */
```

Comment:
Just a note: I think we might still use some of these deprecated types in the tina-cloud-starter. We will have to update the starter once this PR goes though.

Reply:
I deleted it because `TinaSchema` is actually the class so this was actually wrong. I don't see it over in our starter though anyway